tests/e2e/util.go

```diff
@@ -0,0 +1,444 @@
+package e2e
```

Comment:
I probably missed this the first time because GitHub didn't expand the file and i scrolled past it. Sorry. But this one is also missing the K8s header.

Reply:
Added